tensorly/decomposition/_tucker.py

```diff
@@ -76,7 +76,7 @@ def initialize_tucker(tensor, rank, modes, random_state, init='svd', svd='numpy_
     return core, factors
     
 
-def partial_tucker(tensor, modes, rank=None, n_iter_max=100, init='svd', tol=10e-5,
```

Comment:
Previously rank had a default of None, but will now have no default. I think this also better matches other signatures, but want to confirm this is what you intended.